css/store.css

```diff
@@ -1,47 +1,146 @@
-body{
+body {
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+}
+
+.header {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  /* background-color: rgb(223, 219, 219); */
+}
 
+.imageLeft {
+  grid-column: 1/2;
+  /* background-image: url('/img/karma-logo.svg'); */
+}
 
+.karmaIcon {
+  width: 40px;
+  height: 40px;
+  margin-left: 50px;
 }
 
-.header{
+.navBar {
+  grid-column: 2/3;
+  margin-left: 10px;
+  /* padding-left: 100px; */
+  align-self: center;
+  /* text-decoration: none; */
+}
 
-    display: grid;
-    grid-template-columns: repeat(2, 1fr);
+.linkStyle {
+  text-decoration: none;
+  margin-right: 20px;
+}
 
+.nav1 {
+  grid-column: 7/14;
+  color: black;
+}
+
+.nav2 {
+    grid-column: 7/14;
+    color: grey;
+  }
+
+  .nav3 {
+    grid-column: 7/14;
+    color: rgb(252, 109, 57);
+  }
+  
+  .nav4 {
+    grid-column: 7/14;
+    color: gray;
+  }
+  
+  .nav5 {
+    grid-column: 7/14;
+    color: gray;
+  }
+  
+  .nav6 {
+    grid-column: 7/14;
+    color: gray;
+  }
+  
+
+.main {
+  display: grid;
+  grid-template-columns: 10% 30% 10% 50%;
+  grid-template-rows: 1fr 1fr 1fr 1fr 1fr;
 }
 
-.imageLeft{
-    grid-column: 1/2;
-    /* background-image: url('/img/karma-logo.svg'); */
+.formDiv {
+  grid-column: 2/3;
 }
 
-.karmaIcon{
-    width: 40px;
-    height:40px;
+.formh1 {
+  color: rgb(252, 109, 57);
+  font-weight: 400;
+  margin-top: 100px;
+}
 
+.mainImageDiv {
+  grid-column: 4/5;
+  grid-row: 1/6;
+  background-image: url("/level-2/storeImage.jpg");
```

Comment:
I think it would be better you add the image in the html rather than the css.

Reply:
Hmm, that approach was commented over in the html side cuz it wasn't scaling properly